account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -54,11 +55,22 @@ export function alchemyGasManagerMiddleware(
 
 interface AlchemyGasAndPaymasterAndDataMiddlewareParams {
   policyId: string | string[];
+  policyToken?: PolicyToken;
   transport: AlchemyTransport;
   gasEstimatorOverride?: ClientMiddlewareFn;
   feeEstimatorOverride?: ClientMiddlewareFn;
 }
 
+export type PolicyToken = {
+  address: Address;
+  maxTokenAmount?: bigint;
+  approvalMode?: "NONE" | "PERMIT";
+  erc20Name?: string;
+  version?: string;
+};
+
+const MaxUint256: bigint = (1n << 256n) - 1n;
```

Comment:
Convention in TypeScript is to name constants in upper snake-case, e.g. `MAX_UINT_256`.

Reply:
viem also exports `maxUint256` btw